content/01.main_content.md

```diff
@@ -24,10 +26,19 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
   For instance, classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007].
   If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of Iberian individuals, underrepresentation of Latin American scientists would go unnoticed in an exclusively name-based analysis.
 
+Beyond accurate classification of individuals' social identifiers, auditing systems also require a comparable background distribution.
+  For example, in Le et al. [@doi:10.1016/j.cels.2021.07.007], in order to identify disparities in ISCB honorees, they compared against scientists' publication records.
```

Comment:
Is it weird to say "they" when there's overlap in the author lists of that paper and this one? More of a philosophical question than a suggestion we change it.

Reply:
hm yes, I reordered the sentence to remove the need for "they"
https://github.com/greenelab/disparities_opinion/pull/16/commits/a5b3c54b8466e3080e0e6d52c75cbec7f5528f53